src/components/search/SearchInput.tsx

```diff
@@ -4,6 +4,7 @@ import BN from 'bn.js'
 import clsx from 'clsx'
```

Comment:
We have the config field 
https://github.com/dappforce/polkaverse/blob/main/src/config/availableFeatures.ts#L10

So just update it

And check that it works correctly

Reply:
changed the check to `enableSearch` env in [20e8f8a](https://github.com/dappforce/polkaverse/pull/87/commits/20e8f8a2f694c6c035528d2ae5ef3c7a57c60c60)